source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -686,9 +685,35 @@ Specifies the maximun number of files to be monitored.
 +--------------------+------------------------------------------+
 
 
-**registries**
+registry_limit
+--------------
+
+Specifies a limit on the number of registry values that will be monitored by syscheck. Registry values created when the database has reached the limit will be ignored.
```

Comment:
```suggestion
Specifies a limit on the number of registry values that will be monitored by FIM. Registry entries created when the database has reached the limit will be ignored.
```

Reply:
The limit only applies to registry values, I think that it will be confusing to use registry entries